lib/teiserver_web/controllers/admin/user_controller.ex

```diff
@@ -1104,4 +1152,24 @@ defmodule TeiserverWeb.Admin.UserController do
         |> redirect(to: ~p"/teiserver/admin/user")
     end
   end
+
+
+  @spec delete_user(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def delete_user(conn, %{"id" => id}) do
+    user = Account.get_user_by_id(id)
+
+    case Teiserver.Account.UserLib.has_access(user, conn) do
+      {true, _} ->
+        Teiserver.manually_delete_user(id)
```

Comment:
That's my biggest problem in this change. `manually_delete_user` is a mix task, and is supposed to be invoked through the terminal with the mix command, or some script, but definitely not from application code.

There is `Teiserver.Account.delete_user` that is more suitable, although it doesn't remove all the linked rows.
A quick grep shows that it's likely a generated stub and isn't really used. I'd rather have this function taking care of deleting the various entities and then you can call it from there.

Reply:
```manually_delete_user``` uses ```Teiserver.Admin.DeleteUserTask.delete_users``` internally, which I now also use. 
I could also use ```Teiserver.Account.delete_user``` if that is better.